evalml/pipelines/components/component_base.py

```diff
@@ -86,10 +85,8 @@ def fit(self, X, y=None):
             self
         """
         X = infer_feature_types(X)
-        X = _convert_woodwork_types_wrapper(X.to_dataframe())
```

Comment:
Can we update the docstrings to no longer take in ww.DataTable / ww.DataColumn for all the components?

Reply:
Done!